docs/querying/sql-functions.md

```diff
@@ -116,11 +116,29 @@ Counts distinct values of a string, numeric, or `hyperUnique` column using Druid
 
 ## APPROX_COUNT_DISTINCT_DS_HLL
 
-`APPROX_COUNT_DISTINCT_DS_HLL(expr, [<NUMERIC>, <CHARACTER>])`
+Approximates distinct values of an HLL sketch column or a regular column. The [HLL sketch documentation](../development/extensions-core/datasketches-hll.md) describes the optional `lgK` and `tgtHllType` arguments.
```

Comment:
Since you don't introduce the syntax until line 121, it's best not to mention parameters here. Instead, you can do something like this:

See [DataSketches HLL Sketch module](../development/extensions-core/datasketches-hll.md) for a description of optional parameters.